modules/zip/doc_classes/ZIPReader.xml

```diff
@@ -57,5 +57,15 @@
 				Must be called after [method open].
 			</description>
 		</method>
+		<method name="get_file_unix_permissions">
+			<return type="int" enum="FileAccess.UnixPermissionFlags" is_bitfield="true" />
+			<param index="0" name="path" type="String" />
+			<param index="1" name="case_sensitive" type="bool" default="true" />
+			<description>
+				Returns file UNIX permissions.
+				[b]Note:[/b] This method is implemented on Linux/BSD, and macOS.
```

Comment:
Is this true? The code isn't platform specific as far as I can tell

Reply:
As Windows don't use unix permissions, so even if we can run this method on windows, the int returned don't mean anything. And for other platforms i can't test and so i don't want to say that it will work

Reply:
The method is still implemented and would work for files packed with permissions enabled, so it wouldn't be universally true it doesn't work on other platforms

Reply:
I tried to match the FileAccess description, but i decided to not add iOS because i can't be really sure it will work (as i don't own one). That's where i got the list from.

What description do you suggest? Sorry for the mistakes i've made, it's the first time i'm doing a PR

Reply:
It works for any platform, as it reads from the ZIP, not the file system, so unless you have source for it not working on those platforms I'd say to not add it here

No worries on any mistakes :) Just asking questions here and I might be wrong, it seems to me that zip files always support these permissions, but it might be that they won't work on Windows when creating archives, would be good to get some confirmation on that

But this reads directly from the zip file, so it should just match what's inside the zip archive, so if it's created on Windows it might not have valid permissions

So I'd suggest testing it, and basing it on that, possibly with a warning that zip archives created on some platforms won't support this